src/components/landing/GroupLogin.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LandingCard from './LandingCard.jsx';
+import TextInput from '../TextInput.jsx';
+import Button from '../Button.jsx';
+import '../../scss/landing.scss';
+
+/// callBack should be passed from landing page. Used to re-render the landing page
+/// with the default landing page card, which should be the login card.
+
+function GroupLogin({ callBack }) {
+  let input;
+
+  function submitCode(event) {
+    event.preventDefault();
+    // TODO: Make login request
+  }
+
+  return (
+    <LandingCard title="No account? No problem">
+      <form onSubmit={submitCode}>
+        <div className="card-form">
+          <TextInput
+            placeHolder="Group Code"
+            // eslint-disable-next-line no-unused-vars
+            onChange={(c) => { input = c; }}
+          />
+          <Button onClick={submitCode}>
+            Enter Code
+          </Button>
+          <div
```

Comment:
It might be worth adding `cursor: pointer` so the link appears clickable. May be even better to use a button with a transparent background even

Reply:
> It might be worth adding `cursor: pointer` so the link appears clickable. May be even better to use a button with a transparent background even

Maybe that's something that I need to add to the `Button` component scss itself.